src/shell/Shell.cpp

```diff
@@ -322,7 +325,7 @@ static Trap::TrapResult executeWASM(Store* store, const std::string& filename, c
                     },
                     nullptr));
             }
-        } else if (import->moduleName() == "wasi_snapshot_preview1") {
+        } else if (import->moduleName() == "wasi_snapshot_preview1" || import->moduleName() == "wasi") {
```

Comment:
Is the name of `wasi` module an official name?

Reply:
I believe the wabt interpreter uses it as a command line argument
https://github.com/Samsung/walrus/blob/d4aa63f62c1a517f3d02160bf6facc8388a45951/third_party/wabt/src/tools/wasm-interp.cc#L102C14-L102C14
And since both previews are getting deprecated in the future I thought it would be easier to just use `wasi`. It can be removed of course if you prefer that.

Reply:
AFAIK only `wasi_snapshot_preview1` is used as the name of WASI (correct me if wrong).
When we convert `c` source codes to corresponding wasm sources by the emscripten tool, 
wasm sources invoke WASI functions with the name of `wasi_snapshot_preview1`.
We may fix the name later if WASI spec is confirmed.

Reply:
I removed the `wasi` option then and will address this if it is necesarry when the spec is updated. Sadly it is a slow process as I have seen.